web/src/search/input/SearchHelpDropdownButton.tsx

```diff
@@ -28,6 +28,12 @@ export const SearchHelpDropdownButton: React.FunctionComponent = () => {
                 <DropdownItem divider={true} />
                 <DropdownItem header={true}>Finding matches:</DropdownItem>
                 <ul className="list-unstyled px-2 mb-2">
+                    <li>
+                        <span className="text-muted small">Structural:</span>{' '}
+                        <code>
+                            <strong>try&#123;:[my_match]&#125; </strong>
```

Comment:
```suggestion
                            <strong>try&#123;&nbsp;:[my_match]&#125; </strong>
```

^ Minor thing, think a space would make sense, but I didn't check if that works and it's minor.

Reply:
Good suggestion.  I added a space to the other side as well. Here's what it looks like:  
![image](https://user-images.githubusercontent.com/539268/86866133-37593d00-c085-11ea-9f6b-b28334137dbf.png)

Reply:
Oh shoot, I actually meant to put it after the `try` 😆  so it would be `try {:my_match}`. Alternatively, we could do `if (:[my_match])` since I think `if (...)` is a very common syntax, but `try {...}` not so much (it doesn't exist in Go, for example).

Reply:
If is a great suggestion. Making the change.